### DISCUSSION TITLE
p.200 SECTION-074 リスト18.22 後処理のdb.Close()について

### DISCUSSION THREAD

**Discussion by magavel** [budougumi0617/go_todo_app#64] [Ideas]
p.200 SECTION-074 リスト18.22の`t.Cleanup(func() { db.Close() })`は、正しくは`t.Cleanup(func() { _ = db.Close() })`でしょうか？

**store/task_test.go**

```diff
func TestRepository_AddTask(t *testing.T) {

// 省略

-	t.Cleanup(func() {  db.Close() })
+	t.Cleanup(func() { _ = db.Close() })

// 省略

}
```

**Comment by budougumi0617** [comment]
正誤表に反映しました。ご指摘ありがとうございました！
https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.8
